docs/standard/base-types/standard-numeric-format-strings.md

```diff
@@ -27,7 +27,7 @@ Standard numeric format strings are used to format common numeric types. A stand
   
 -   `xx` is an optional integer called the *precision specifier*. The precision specifier ranges from 0 to 99 and affects the number of digits in the result. Note that the precision specifier controls the number of digits in the string representation of a number. It does not round the number itself. To perform a rounding operation, use the <xref:System.Math.Ceiling%2A?displayProperty=nameWithType>, <xref:System.Math.Floor%2A?displayProperty=nameWithType>, or <xref:System.Math.Round%2A?displayProperty=nameWithType> method.  
   
-     When *precision specifier* controls the number of fractional digits in the result string, the result strings reflect numbers that are rounded away from zero (that is, using <xref:System.MidpointRounding.AwayFromZero?displayProperty=nameWithType>).  
+     When *precision specifier* controls the number of fractional digits in the result string, the result string reflects a number that is rounded to a representable result nearest to the infinitely precise result, or the result with an even least significant digit if there are two equally near representable results. (that is, using <xref:System.MidpointRounding.ToEven?displayProperty=nameWithType>).  
```

Comment:
I'd rephrase this to reflect the differences between .NET Framework, .NET Core 2.0, and .NET Core 2.1:

- When *precision specifier* controls the number of fractional digits in the result string, the result string reflects a number that is rounded to a representable result nearest to the infinitely precise result. If there are two equally near representable results:
   - \*\*On the .NET Framework and .NET Core up to .NET Core 2.0\*\*, the result with the greater least significant digit (that is, using \<xref:System.MidpointRounding.ToEven?displayProperty=nameWithType>).
   - \*\*On .NET Core 2.1 and later\*\*, the result with an even least significant digit (that is, using \<xref:System.MidpointRounding.ToEven?displayProperty=nameWithType>).

Reply:
Updated